demonstrations/tutorial_classical_kernels.py

```diff
@@ -0,0 +1,491 @@
+r"""
+
+.. _classical_kernels:
+
+Emulating classical kernels
+============================
+
+.. meta::
+    :property="og:description": Show how to approximate the Gaussian kernel.
```

Comment:
```suggestion
    :property="og:description": Approximating the Gaussian kernel with quantum circuits.
```
Just a suggestion...